bin/pathway_analysis.R

```diff
@@ -153,45 +160,46 @@ for (file in contrast_files){
                   sources=datasources,
                   evcodes=TRUE,
                   user_threshold=0.05)
-
+  
   # Make data frame of gost result
   pathway_gostres <- gostres$result
   # Select only significantly enriched pathways (according to adjusted p-value)
   pathway_gostres <- as.data.frame(pathway_gostres[which(pathway_gostres$significant==TRUE),])
-
+  pathway_gostres <- pathway_gostres[which(pathway_gostres$intersection_size>=min_DEG_pathway),]
   # Plot pathways if there were any
   if (nrow(pathway_gostres) > 0){
-
+    
     # annotate query size (number of DE genes in contrast)
     pathway_gostres$original_query_size <- rep(length(q), nrow(pathway_gostres))
 
+    
     # Generate non-interactive pathway dotplots in the folder
     pg <- gostplot(gostres, capped=T, interactive=F)
```

Comment:
As here it plots the `gostres` again, which has not been modified. Do you need to edit here also `gostres$results` to also edit this plots?